migrations/20231025145134_add_notification.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TYPE notification_states
+  AS ENUM ('queued', 'processing', 'published', 'not-subscribed', 'wrong-scope', 'rate-limited');
+
+CREATE TABLE notification (
+    id         UUID         PRIMARY KEY DEFAULT gen_random_uuid(),
+    created_at TIMESTAMPTZ  NOT NULL DEFAULT now(),
+    type       VARCHAR(255) NOT NULL,
+    title      VARCHAR(255) NOT NULL,
+    body       VARCHAR(255) NOT NULL,
+    icon       VARCHAR(255) NOT NULL,
+    url        VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE notification_delivery (
+    id            UUID                 PRIMARY KEY DEFAULT gen_random_uuid(),
+    created_at    TIMESTAMPTZ          NOT NULL DEFAULT now(),
+    updated_at    TIMESTAMPTZ          NOT NULL DEFAULT now(),
+    status        notification_states  NOT NULL,
+    notification_id  UUID              NOT NULL REFERENCES notification (id) ON DELETE CASCADE,
+    subscriber_id    UUID              NOT NULL REFERENCES subscriber (id) ON DELETE CASCADE
```

Comment:
Other tables don't have `_id` suffix

```suggestion
    notification     UUID              NOT NULL REFERENCES notification (id) ON DELETE CASCADE,
    subscriber       UUID              NOT NULL REFERENCES subscriber (id) ON DELETE CASCADE
```

Reply:
I'll change it for the consistency. Thx.

Reply:
Done.